### PR TITLE
Data access services

### PR DESCRIPTION
**Pull request by AlexanderSerbin**

### CODE REVIEW

File: legacy-data-access-services/src/main/java/gov/ca/cwds/cms/data/access/service/DataAccessService.java [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+/**
+ * @author CWDS CALS API Team
+ */
+
+public interface DataAccessService<T extends PersistentObject, P extends BaseParameterObject> {
+
+  T create(T entity, P parameterObject);
+
+  default void runBusinessRules(T entity) {
```
Comment (vladimirPe) [inline legacy-data-access-services/src/main/java/gov/ca/cwds/cms/data/access/service/DataAccessService.java]: May be runBusinessValidation is better naming for it

File: legacy-data-access-services/src/main/java/gov/ca/cwds/cms/data/access/service/impl/PlacementHomeServiceImpl.java [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+  }
+
+  private void storeCountyOwnership(String placementHomeId) {
+    CountyOwnership countyOwnership =
+        countyOwnershipMapper.toCountyOwnership(placementHomeId,
+            "P", Collections.emptyList());
+    countyOwnershipDao.create(countyOwnership);
+  }
+
+  private void storePlacementHomeUc(PlacementHome placementHome,
```
Comment (vladimirPe) [inline legacy-data-access-services/src/main/java/gov/ca/cwds/cms/data/access/service/impl/PlacementHomeServiceImpl.java]: IMHO a mapper class  is better for mapping

Comment (AlexanderSerbin) [reply legacy-data-access-services/src/main/java/gov/ca/cwds/cms/data/access/service/impl/PlacementHomeServiceImpl.java]: Thank you for your comment. I appreciate your point of view

File: legacy-data-access-services/src/main/java/gov/ca/cwds/cms/data/access/service/DataAccessService.java (hunk 1 above)
Comment (AlexanderSerbin) [reply legacy-data-access-services/src/main/java/gov/ca/cwds/cms/data/access/service/DataAccessService.java]: Agree. Will rename it

**Pull request merged**